libseccomp/src/arch.rs

```diff
@@ -25,6 +25,10 @@ pub enum ScmpArch {
     Arm,
     /// The AARCH64 architecture token
     Aarch64,
+    /// The LOONGARCH64 architecture token
+    Loongarch64,
+    /// The M68K architecture token
```

Comment:
`The Motorola 68000 architecture token` is better.

Ref. https://github.com/seccomp/libseccomp/blob/main/include/seccomp.h.in#L168